neighsyncd/neighsync.cpp

```diff
@@ -70,6 +70,11 @@ void NeighSync::onMsg(int nlmsg_type, struct nl_object *obj)
     key+= ipStr;
 
     int state = rtnl_neigh_get_state(neigh);
+    if (state == NUD_NOARP)
```

Comment:
It is related. In kernel the neighbor entry would be generated on vrf interface. It wouldn't notify netlink message. But it can get by `dumpRequest`. So when warm-reboot the neighbor point to vrf interface will get back and they cannot be deal with by neighorch. This type of neighbor on l3mdev or loopback_dev don't need to handle, which state is `NUD_NOARP`. Actually when no vrf, the `lo` is similar and filtered by neighorch.

Reply:
ok